components/admin_console/license_settings/license_utils/license_utils.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import AlertBanner from 'components/alert_banner';
+import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+
+import {localizeMessage} from 'utils/utils';
+
+export const free30DayTrial = (
+    isDisabled: boolean,
+    gettingTrialError: JSX.Element | null,
+    requestLicense: (e: any) => Promise<void>,
+    gettingTrial: boolean,
+): JSX.Element => {
+    return (
+        <AlertBanner
+            mode='info'
+            title={
+                <FormattedMessage
+                    id='licensingPage.infoBanner.startTrialTitle'
+                    defaultMessage='Free 30 day trial!'
+                />
+            }
+            message={
+                <div className='banner-start-trial'>
+                    <p className='trial-legal-terms'>
```

Comment:
Can we double check what font size we are using here? It seems the text is much smaller that the other cards and in the desing it should be similar - this is how it looks now: 
<img width="982" alt="Screen Shot 2021-11-14 at 9 06 45 PM" src="https://user-images.githubusercontent.com/936315/141717918-79652a61-4f07-4f25-9e65-cc192e346d9a.png">